io/plugins/eu.esdihumboldt.hale.io.gml/src/eu/esdihumboldt/hale/io/gml/writer/GmlInstanceWriter.java

```diff
@@ -45,4 +46,12 @@ protected boolean requiresDefaultContainer() {
 		return true; // requires a FeatureCollection being present
 	}
 
+	/**
+	 * @see eu.esdihumboldt.hale.common.instance.io.impl.AbstractGeoInstanceWriter#getDefaultWindingOrder()
+	 */
+	@Override
+	protected EnumWindingOrderTypes getDefaultWindingOrder() {
+		return EnumWindingOrderTypes.counterClockwise;
```

Comment:
`StreamGmlWriter` is the base class of GML writers, please move the method there.